File: Baekjoon_Online_Judge/NO_09664/9664.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 07                   | #
# | 9664 NASLJEDSTVO         | #
# ---------------------------- #


if __name__ == "__main__":
    N = int(r_input())
    O = int(r_input())

    O *= N / (N - 1)
    
    flag = int(O % 1 == 0)
    O = int(O)

    print(O - flag, O)

File: Baekjoon_Online_Judge/NO_09664/9664.c
#include <stdio.h>
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 07                   | 
 | 9664 NASLJEDSTVO         | 
 ---------------------------- 
*/

int main(){
    int N, O, mini = 2147483647, maxi = 0;

    scanf("%d %d", &N, &O);

    for(int num = 1; num <= O * 2; num ++){
        if(num - num / N == O){
            if(mini > num) mini = num;
            if(maxi < num) maxi = num;
        }
    }

    printf("%d %d", mini, maxi);

    return 0;
}
